src/meshdb/utils/spreadsheet_import/fetch_uisp.py

```diff
@@ -29,6 +30,18 @@ def download_uisp_links():
     )
 
 
+def get_uisp_device_detail(device_id: str, session: Optional[requests.Session] = None) -> dict:
```

Comment:
Add a comment about the kind of data we get or maybe elaborate the `dict`?

Reply:
"meh"
- Andrew, 2024

Reply:
Nah UISP sucks